.eslintrc.yaml

```diff
@@ -0,0 +1,71 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
+#
+# Be sure to add your project's specific environment(s) as needed via external configuration.
+#   See: http://eslint.org/docs/user-guide/configuring#specifying-environments
+
+---
+rules: # http://eslint.org/docs/rules
+
+  # Possible Errors
+
+  valid-jsdoc: [1]
+  curly: [2]
+  dot-notation: [2]
+  eqeqeq: [2]
+  no-div-regex: [2]
+  no-eval: [2]
+  no-extend-native: [2]
+  no-floating-decimal: [2]
+  no-lone-blocks: [2]
+  no-loop-func: [2]
+  no-multi-spaces: [2]
+  no-native-reassign: [2]
+  no-new-wrappers: [2]
+  no-return-assign: [2]
+  no-self-compare: [2]
+  radix: [2]
+  yoda: [2, "never"]
+
```

Comment:
recommended extras: [guard-for-in](http://eslint.org/docs/rules/guard-for-in) (ideally, not allowing for-in at all, since it's a dangerous bit of 1998 low level JS, the modern equivalent is Object.keys(), for **on** (very different!) and forEach iteration, instead...)